--- simgear-2019.1.2/CMakeLists.txt.orig	2020-02-21 10:45:47.189449585 +0200
+++ simgear-2019.1.2/CMakeLists.txt	2020-02-21 10:46:41.236943284 +0200
@@ -14,6 +14,8 @@
 
 
 message(STATUS "CMAKE Build type: ${CMAKE_BUILD_TYPE}")
+
+include_directories( AFTER "${CMAKE_CURRENT_BINARY_DIR}" )
 # Set a default build type if none was specified
 if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
   message(STATUS "Setting build type to 'Debug' as none was specified.")
@@ -107,8 +109,8 @@
 
 if (NOT MSVC)
 option(SIMGEAR_SHARED   "Set to ON to build SimGear as a shared library/framework" OFF)
-option(SYSTEM_EXPAT     "Set to ON to build SimGear using the system expat library" OFF)
-option(SYSTEM_UDNS      "Set to ON to build SimGear using the system udns library" OFF)
+option(SYSTEM_EXPAT     "Set to ON to build SimGear using the system expat library" ON)
+option(SYSTEM_UDNS      "Set to ON to build SimGear using the system udns library" ON)
 else()
 # Building SimGear DLLs is currently not supported for MSVC.
 set(SIMGEAR_SHARED OFF)
